util/translate_timeline.ts

```diff
@@ -4,7 +4,7 @@ import { ArgumentParser } from 'argparse';
 import { findMissing } from './find_missing_timeline_translations';
 import { Event, Sync, Timeline } from '../ui/raidboss/timeline';
 import { walkDirSync } from './file_utils';
-import { Lang } from 'types/global';
+import { Lang } from 'resources/languages';
```

Comment:
I think should this be a relative path like 

```suggestion
import { Lang } from '../resources/languages';
```

Reply:
I tried to keep the relative vs absolute paths the same as they were before. I can adjust them if needed?

Reply:
I think this is fine to keep for now, but I think it'd be good to go back and be consistent (and eslint) in a followup PR?

Reply:
There are spots all over the place that would need to be updated for an eslint to pass, I think. I can work on a PR for it, looks like 61 cases in 26 files according to a quick regex search. Sounds like something for me to do during work meetings tomorrow.

Reply:
Is there an eslint rule that could prevent this?

Reply:
I think `no-restricted-imports` with a pattern that only allows path starts with "./" or "../" would help?

Reply:
This should work, though I'll have to test it at some point.

```js
"no-restricted-imports": ["error", {
    "patterns": ["!./*"]
}]
```

Reply:
no, not working

Reply:
I've got a working version but it needs things excluded manually if they're globals or whatever. I've excluded the core ts imports, for example, and it's still giving:

```
C:\Users\valar\Source\cactbot2\util\process_triggers_folder.ts
  2:1  error  'eslint' import is restricted from being used by a pattern                           no-restricted-imports
  5:1  error  'process' import is restricted from being used by a pattern                          no-restricted-imports
  6:1  error  './file_utils' import is restricted from being used by a pattern                     no-restricted-imports
  7:1  error  '../resources/user_config' import is restricted from being used by a pattern         no-restricted-imports
  8:1  error  '../ui/raidboss/raidboss_options' import is restricted from being used by a pattern  no-restricted-imports
```

The rule at this point is already getting fairly big.

```js
  'no-restricted-imports': [
    'error',
    {
      'patterns': [
        // Include all imports
        '*',
        // Exclude relative imports
        '!./*', '!../*',
        // Exclude core imports
        '!assert',
        '!buffer',
        '!child_process',
        '!cluster',
        '!crypto',
        '!dgram',
        '!dns',
        '!domain',
        '!events',
        '!freelist',
        '!fs',
        '!http',
        '!https',
        '!module',
        '!net',
        '!os',
        '!path',
        '!punycode',
        '!querystring',
        '!readline',
        '!repl',
        '!smalloc',
        '!stream',
        '!string_decoder',
        '!sys',
        '!timers',
        '!tls',
        '!tracing',
        '!tty',
        '!url',
        '!util',
        '!vm',
        '!zlib'
      ],
    }
  ],
```

Reply:
(I realize that it's not 100% working yet, I'm still tweaking, this is just an example.)

Reply:
@quisquous I don't think this should hold up this PR, the discussion can keep happening here or be moved to an issue?